doc/api/process.md

```diff
@@ -2305,6 +2305,222 @@ import { loadEnvFile } from 'node:process';
+   ```
+
+2. **Comments**:
+
+   * Lines starting with `#` are treated as comments and ignored.
+   * Inline comments (starting with `#` after a value) are ignored, and do not
+     affect parsing of the value.
+
+   ```bash
+       # COMMENTS=work
+       INLINE_COMMENTS=inline comments # work
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.COMMENTS, undefined);
+   assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.COMMENTS, undefined);
+   assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+   ```
+
+3. **Whitespace handling**:
+
+   * Leading and trailing whitespaces around keys and values are ignored unless
+     they are enclosed within quotes.
+
+   ```bash
+         SPACED_KEY = parsed
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+   ```
+
+4. **Empty values**:
+
+   * Variables with no value assigned (just a key followed by `=`) are set to
+     an empty string.
+
+   ```bash
+     EMPTY=
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.EMPTY, '');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.EMPTY, '');
+   ```
+
+5. **Quoted values**:
+
+   * Single (`'`), double (`"`), and backtick (`` ` ``) quotes are recognized.
+     The content within the quotes is taken as is, including leading and
+     trailing spaces.
+   * Quotes within a different quote type are preserved.
+
+   ```bash
+     FIRST_NAME=' John '
+     MIXED_QUOTES="Say 'Hello!'"
+     BACKTICK_IN_QUOTES="Using `backticks` in double quotes"
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+6. **Multiline values**:
+
+   * Values enclosed in double, single, or backtick quotes that span multiple
```

Comment:
```suggestion
   * Values enclosed in quotes described above that span multiple
```

Reply:
It may also make sense to mention that unlike shells `"` and `'` behave identially. In shells _typically_ `'` doesn't interpolate anything, " allows inline interpolation and backticks execute a command in a subshell.